test/onnx/expect/TestOperators.test_layer_norm_aten.expect

```diff
@@ -111,16 +112,16 @@ graph {
         elem_type: 1
         shape {
           dim {
-            dim_value: 20
```

Comment:
Are these changes from static shape to dynamic shape intended and expected?

Reply:
That I can't tell. It makes no sense to me considering the nature of changes that I have done. 

What I would guess is that shape inference for aten domain is somehow different than for onnx domain?

Reply:
> What I would guess is that shape inference for aten domain is somehow different than for onnx domain

Makes sense. @jiafatom should know more. It might have something to do with `TypeReliable` etc.

Reply:
This change still looks suspicious as it's not only one test affected. I'm not familiar with ONNX export shape inference, but it feels like some results became dynamic.

Reply:
For onnx domain, shape inference may go to the inference mode, and then `TypeReliable` would be `true`. For aten domain, shape inference is obtained through tracing, so `TypeReliable` would be `false` unless the operator is defined in `static std::unordered_set<std::string> nodeTypeReliableForTracer`. When `TypeReliable` is `false`, it would cause a dynamic shape, that seems the reason here.

Reply:
I have confirmed this by adding `aten::ATen` to the list of reliable types, restoring the original behavior (which might be safer, but less accurate)